scripts/install_and_enable_addons_render.py

```diff
@@ -0,0 +1,519 @@
+"""
+To run in Blender from the terminal:
+    blender
+    --python <path to this script>
+    -- <space-separated list of paths to the addons to enable>
+
+Example:
+    blender
+    --python ./blender_randomiser/install_and_enable_addons.py
+    -- ./blender_randomiser/add_array_objects_to_cursor.py
+    ./blender_randomiser/add_random_cube_in_volume.py
+
+
+
+"""
+
+
+# %% Libs
+import argparse
+import json
+import re
+import sys
+from pathlib import Path
+from random import seed
+
+import bpy
+
+
+# %% Main function
+def main():
+    """
+    User defined:
+    --seed:     Randomisaiton Seed
+    --input:    Input json file
+    --output:   Output json file
+    --render_seg_only:   Render the segmentation only
+    --render_main_only:   Render the main image only
```

Comment:
What is the main image?

If you mean the RGB image maybe this can be called RGB only